sunpy/net/jsoc/jsoc.py

```diff
@@ -618,18 +618,16 @@ def get_request(self, requests, path=None, overwrite=False, progress=True,
                     fname = os.path.expanduser(fname)
                     paths.append(fname)
 
-        if max_conn * kwargs['max_splits'] > 10:
-            warnings.warn(("JSOC does not support more than 10 parallel connections. " +
-                           "Changing the number of parallel connections to 8."), SunpyUserWarning)
-            kwargs['max_splits'] = 2
-            max_conn = 4
-
         dl_set = True
         if not downloader:
             dl_set = False
             downloader = Downloader(progress=progress, overwrite=overwrite, max_conn=max_conn)
-        else:
-            downloader.max_conn = max_conn
+
+        if max_conn * kwargs['max_splits'] > 10:
```

Comment:
Do you not need:
```suggestion
        if downloader.max_conn * kwargs['max_splits'] > 10:
```

Reply:
Yes, good catch. I'm not really sure why `fetch()` has a `max_conn` argument if we're just passing downloaders with it set? Maybe it's a hangover from pre-parfive times?